tools/test/examples/examples.json

```diff
@@ -463,6 +463,22 @@
         "baud_rate": 9600,
         "compare_log": ["mbed-os-example-attestation/tests/attestation.log"],
         "auto-update" : true
+    },
+    {
+        "name": "mbed-os-example-atecc608a",
+        "github": "https://github.com/ARMmbed/mbed-os-example-atecc608a",
+        "mbed": [],
+        "test-repo-source": "github",
+        "features" : [],
+        "targets" : ["K64F"],
```

Comment:
That is might depends on which target in the RaaS got the `atecc608a` shield. Also it is true we don't want to compile the exampled for too many target, it will not slow down the testing process